README.md

```diff
@@ -1,151 +1,49 @@
-
-The code is able to run on Linux server side, however the libraries for MySQL and logging do require extra packages.
-
-For MySQL, run the following: `apt-get install libmysqlclient-dev:i386`
-
-For RustG, run the following: `apt-get install libssl-dev:i386 pkg-config:i386 zlib1g-dev:i386`
-
-After installing these packages, these libraries should function as intended.
-
----
-
-### UPDATING
-
-To update an existing installation, first back up your /config and /data folders
-as these store your server configuration, player preferences and banlist.
-
-If you used the zip method,
-you'll need to download the zip file again and unzip it somewhere else,
-and then copy the /config and /data folders over.
-
-If you used the git method, you simply need to type this in to git bash:
-
-    git pull
-
-When this completes, copy over your /data and /config folders again, just in case.
-
-When you have done this, you'll need to recompile the code, but then it should work fine.
-
----
-
-### Configuration
-
-For a basic setup, simply copy every file from config/example to config.
-
----
-
-### SQL Setup
-
-The SQL backend for the library and stats tracking requires a MySQL server.
-Your server details go in /config/dbconfig.txt,
-and the SQL schema is in /SQL/paradise_schema.sql or /SQL/paradise_schema_prefix.sql,
-depending on if you want table prefixes.
-More detailed setup instructions are located on our wiki:
-https://www.paradisestation.org/wiki/index.php/Setting_up_the_Database
-
----
-
-### IRC Bot Setup
-
-Included in the repo is an IRC bot capable of relaying adminhelps to a specified IRC
-channel/server (thanks to Skibiliano).
-Instructions for bot setup are included in the /bot/ folder,
-along with the bot/relay script itself.
-
-### LICENSE
-
-Paradise is licensed under the GNU Affero General Public License version 3.
-As of 5th January 2015 any new contributions are licensed under the AGPL as well,
-if you wish to submit code under the GPL v3 then commits and files must be marked as such
-in comments. If you wish to use our code in a closed source manner you may use anything
-before commit 445615b8439bf606ff204a42c8e7b6b69d983255,
-which is licensed under GPL v3.
-The major change here is that if you host a server using any code licensed under AGPL you
-are required to provide full source code for your servers users as well,
-including addons and modifications you have made.
-
-See [this](https://www.gnu.org/licenses/why-affero-gpl.html) for more information.
-
-Any files located in the
-`Paradise/goon`,
-`Paradise/icons/goonstation`, or
-`Paradise/sound/goonstation`
-directories, or any subdirectories of mentioned directories are licensed under the
-Creative Commons 3.0 BY-NC-SA license
-(https://creativecommons.org/licenses/by-nc-sa/3.0)
-
-All other assets including icons and sound files are licensed under the
-Creative Commons 3.0 BY-SA license (https://creativecommons.org/licenses/by-sa/3.0/),
-unless otherwise indicated.
+# Paradise
```

Comment:
I appear to have mass changed line endings, but now is probably the best time to make this file LF instead of CRLF